Sources/HaCWebsiteLib/Events/EventManager.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+public struct EventManager {
+    private(set) static var events: [Event] = []
+
+    public static func update() throws{
+        var updatedEvents : [Event] = []
+
+        try! WorkshopManager.update()
+        //TODO: source events from database
+        let workshopEvents : [Event] = WorkshopManager.workshops.map{
+            (workshop: Workshop) -> WorkshopEvent in
+                return WorkshopEvent(called: workshop.title, at: Date(), described: "There's Pizza", 
+                    coloured: "purple", hypePeriod: 7, coolOffPeriod: 7, basedOn: workshop)
+        }.filter{ event in 
+            event.isLive()
+        }
+        updatedEvents += workshopEvents
+
+        updatedEvents += [HackathonEvent(called: "Game Gig 2017", at: Date(), described: "Fun and Games", 
+            coloured: "black", hypePeriod: 7, coolOffPeriod: 7, lasting: 3)]
+
+        events = updatedEvents
+    }
+}
+
+extension Event {
```

Comment:
Nice use of extensions here. Do we want these to be private so they don't leak out into the rest of the project?